cg/services/analysis_starter/configurator/models/mip_dna.py

```diff
@@ -1,14 +1,36 @@
 from pydantic import Field
 
 from cg.constants.constants import Workflow
+from cg.constants.priority import SlurmQos
 from cg.services.analysis_starter.configurator.abstract_model import CaseConfig
 
 
 class MIPDNACaseConfig(CaseConfig):
-    bwa_mem: int | None = None
-    bwa_mem2: int | None = None
+    conda_binary: str
+    conda_environment: str
     email: str
-    slurm_qos: str
+    pipeline_binary: str
+    pipeline_config_path: str
+    slurm_qos: SlurmQos
```

Comment:
remember to sync this naming with microSALT (or rename microSALT in agreement with everybody, still to discuss)